### PR TITLE
add donation form presenter

### PR DESCRIPTION
**Pull request by KaiNissen**
This introduces a donation form presenter. Data that was already entered using a banner is being passed to it and made available to the template. Currently, it only processes payment type, amount and payment interval.

### CODE REVIEW

File: app/routes.php [hunk 1]
```diff
@@ -455,11 +457,15 @@ function ( Application $app, Request $request ) use ( $ffFactory ) {
 	}
 );
 
-$app->get( '/', function ( Application $app ) {
-	return $app->handle(
-		Request::create( '/page/DonationForm', 'GET' ),
-		HttpKernelInterface::SUB_REQUEST
-	);
+$app->get( '/', function ( Application $app, Request $request ) use ( $ffFactory ) {
```
Comment (gbirke) [inline app/routes.php]: Is this change really needed? Would't it be handled by the subrequest to add donation?

Comment (KaiNissen) [inline app/routes.php]: The `page` route can't pass those values into the template. Also, a subrequest does not automatically pass on variables sent by `GET` or `POST`.

File: app/routes.php [hunk 2]
```diff
@@ -455,11 +457,15 @@ function ( Application $app, Request $request ) use ( $ffFactory ) {
...
+	$params = $request->query;
+	$amount = Euro::newFromFloat( ( new AmountParser( 'de_DE' ) )->parseAsFloat( $params->get( 'betrag', '' ) ) );
+	$donationRequest = new AddDonationRequest();
```
Comment (JeroenDeDauw) [inline app/routes.php]: Why use the request model for the AddDonation UC here?
